apps/services/user-profile/src/app/user-profile/verification.service.ts

```diff
@@ -138,11 +142,21 @@ export class VerificationService {
       }
     }
 
+    if (verification.tries >= maxTries) {
+      return {
+        message:
+          'Too many failed email verifications. Please restart verification.',
+        confirmed: false,
```

Comment:
To make it easier for the client to handle both this and 0 remaining attempts. We should provide here the field `remainingAttempts: 0`